src/pages/FrontPage.js

```diff
@@ -9,14 +9,16 @@ import TeacherInfobox from '../components/FrontPage/TeacherInfobox';
 import WelcomeBox from '../components/FrontPage/WelcomeBox';
 
 export const  FrontPage = React.createClass({
-
+  componentDidMount() {
+    document.title = 'Kodeklubben';
```

Comment:
Hm, kanskje ikke "i stedet for", greit å endre title for de dynamiske sidene også, men du bør vel uansett endre title i webpack.config.babel.js.